web/components/feed/feed-answer-comment-group.tsx

```diff
@@ -16,44 +14,74 @@ import { Linkify } from 'web/components/linkify'
 import clsx from 'clsx'
 import { tradingAllowed } from 'web/lib/firebase/contracts'
 import { BuyButton } from 'web/components/yes-no-selector'
-import { FeedItem } from 'web/components/feed/feed-items'
 import {
   CommentInput,
+  CommentRepliesList,
   getMostRecentCommentableBet,
 } from 'web/components/feed/feed-comments'
 import { CopyLinkDateTimeComponent } from 'web/components/feed/copy-link-date-time'
 import { useRouter } from 'next/router'
+import { groupBy } from 'lodash'
+import { User } from 'common/user'
 
 export function FeedAnswerCommentGroup(props: {
   contract: any
+  user: User | undefined | null
   answer: Answer
-  items: ActivityItem[]
-  type: string
-  betsByCurrentUser?: Bet[]
-  commentsByCurrentUser?: Comment[]
+  comments: Comment[]
+  bets: Bet[]
 }) {
-  const { answer, items, contract, betsByCurrentUser, commentsByCurrentUser } =
-    props
+  const { answer, contract, comments, bets, user } = props
   const { username, avatarUrl, name, text } = answer
   const answerElementId = `answer-${answer.id}`
-  const user = useUser()
-  const mostRecentCommentableBet = getMostRecentCommentableBet(
-    betsByCurrentUser ?? [],
-    commentsByCurrentUser ?? [],
-    user,
-    answer.number + ''
+  const betsByUserId = groupBy(bets, (bet) => bet.userId)
+  const commentsByUserId = groupBy(comments, (comment) => comment.userId)
+  const [replyToUsername, setReplyToUsername] = useState('')
+  const answerComments = comments.filter(
+    (comment) => comment.answerOutcome === answer.number.toString()
+  )
+  const commentReplies = comments.filter(
+    (comment) =>
+      comment.replyToCommentId &&
+      !comment.answerOutcome &&
+      answerComments.map((c) => c.id).includes(comment.replyToCommentId)
   )
+  const commentsList = answerComments.concat(commentReplies)
+
   const prob = getDpmOutcomeProbability(contract.totalShares, answer.id)
   const probPercent = formatPercent(prob)
   const [open, setOpen] = useState(false)
   const [showReply, setShowReply] = useState(false)
```

Comment:
I would add some new lines around these useState hooks for readability. Consider also grouping them with the other useState above, with `replyToUsername`.